hugegraph-server/hugegraph-api/src/main/java/org/apache/hugegraph/server/ApplicationConfig.java

```diff
@@ -44,16 +44,25 @@
 import io.swagger.v3.oas.annotations.OpenAPIDefinition;
 import io.swagger.v3.oas.annotations.info.Contact;
 import io.swagger.v3.oas.annotations.info.Info;
+import io.swagger.v3.oas.annotations.enums.SecuritySchemeType;
+import io.swagger.v3.oas.annotations.security.SecurityRequirement;
+import io.swagger.v3.oas.annotations.security.SecurityScheme;
 import jakarta.ws.rs.ApplicationPath;
 
+@SecurityScheme(
+    name = "basic",
+    type = SecuritySchemeType.HTTP,
+    scheme = "basic"
+)
```

Comment:
TODO: if users need `token` mode in future, we could add the option like this (now we just enable basic auth)

![68c58c71fa3454a84c07963a8aaaa8a0](https://github.com/apache/incubator-hugegraph/assets/17706099/cd91dd13-5ff9-45ea-931b-5666353f9777)

Reply:
can we keep the basic and token at the same time

Reply:
Done
![image](https://github.com/apache/incubator-hugegraph/assets/48077841/52fdb747-c8a1-492c-a133-9b28cc9771a6)